libevmcore/EVMSchedule.h

```diff
@@ -108,6 +109,7 @@ static const EVMSchedule EIP158Schedule = []
 static const EVMSchedule MetropolisSchedule = []
 {
 	EVMSchedule schedule = EIP158Schedule;
+	schedule.blockhashGas = 800;
```

Comment:
And I will ask in the allcoredev meeting https://github.com/ethereum/pm/issues/14#issuecomment-302364003